components/radio/style/index.less

```diff
@@ -6,6 +6,8 @@
 @radio-inner-prefix-cls: ~'@{radio-prefix-cls}-inner';
 @radio-duration: 0.3s;
 @radio-focused-outline: 3px solid fade(@radio-dot-color, 6%);
+@radio-wrapper-margin-right: 8px;
+@radio-checked-inner-scaling-value: 0.875;
```

Comment:
Look good to me with a nit: personally I prefer `@radio-checked-inner-scale` for conciseness, and consistency with the naming convention [here](https://github.com/ant-design/ant-design/blob/3.15.0/components/style/mixins/iconfont.less#L33).

Reply:
Updated